cypress/e2e/tests/rbac/architect.test.ts

```diff
@@ -18,13 +17,16 @@ import { Stakeholders } from "../../models/migration/controls/stakeholders";
 import { Assessment } from "../../models/migration/applicationinventory/assessment";
 import * as data from "../../../utils/data_utils";
 
+/**
+ * This test set covers validation of different permissions for user with Architect role.
+ */
 describe(["@tier2"], "Architect RBAC operations", () => {
-    let userArchitect = new UserArchitect(getRandomUserData());
+    let adminUserName = Cypress.env("user");
+    let adminUserPassword = Cypress.env("pass");
```

Comment:
*question:* login method already gets the user and password from Cypress config if they're not provided, I'm not sure we have to fetch them also here.

If it's because a previous login is already in cache, it should be enough if we empty the local storage, right?